src/storage/src/hummock/cache.rs

```diff
@@ -543,16 +543,19 @@ pub struct LruCache<K: LruKey, T: LruValue> {
     shard_lru_usages: Vec<Arc<AtomicUsize>>,
 }
 
+// we only need a small object pool because when the cache reach the limit of capacity, it will
+// always release some object after insert a new block.
+const DEFAULT_OBJECT_POOL_SIZE: usize = 1024;
+
 impl<K: LruKey, T: LruValue> LruCache<K, T> {
-    pub fn new(num_shard_bits: usize, capacity: usize, object_cache: usize) -> Self {
+    pub fn new(num_shard_bits: usize, capacity: usize) -> Self {
         let num_shards = 1 << num_shard_bits;
         let mut shards = Vec::with_capacity(num_shards);
         let per_shard = capacity / num_shards;
-        let per_shard_object = object_cache / num_shards;
         let mut shard_usages = Vec::with_capacity(num_shards);
         let mut shard_lru_usages = Vec::with_capacity(num_shards);
         for _ in 0..num_shards {
-            let shard = LruCacheShard::new(per_shard, per_shard_object);
+            let shard = LruCacheShard::new(per_shard, DEFAULT_OBJECT_POOL_SIZE);
```

Comment:
Shall we adjust the pool size by the capacity per shard?

Reply:
It seems not necessary because the object pool is very small...